### ISSUE TITLE
Flywheel failed to load correctly [1.20.1]

### ISSUE THREAD

**Issue by ghostiebailey** [Engine-Room/Flywheel#221]
### Describe the Bug

Tried to load this originally with farmers delight, addons for farmer's delight, and then some addons for create. I've disabled all addons plus disabled Farmer's Delight to narrow it down to this.

NOTE: Version is 1.20.1 and the create version that has built-in flywheel apparently, the version for create is **0.5.1**, and the flywheel version is 6.10, if that helps, but the issue is flywheel.

### Reproduction Steps

1. Run Create for 1.20.1 (0.5.1) (alternatively, run with modpack without create addons or farmer's delight)
2. Start game
3. Crash recieved
...

### Expected Result

After having someone else also run the exact pack before me, watching them succeed in getting into the game and get into a singleplayer world, I was expecting it to run fine. However, I was the only one to have the issue that is occuring. After reinstalling both the pack and the mod, the issue persisted.

### Screenshots and Videos

![2024-01-29 01_30_41-Minecraft Forge_ 1 20 1](https://github.com/Jozufozu/Flywheel/assets/158023163/df202c2b-f438-4110-814a-fe04539ed425)

### Crash Report or Log

https://pastebin.com/S4nMgzGt

### Operating System

Windows 10

### What is your GPU?

NVIDIA GeForce GTX 1050 Ti

### Mod Version

0.5.1

### Minecraft Version

1.19.2

### Loader Version

Forge 47.2.0

### Other Mods

Ad Astra
Ad Astra : Proxima Plus
Ad Astra Extra
Ad Extendra
Apothic Attributes (AttributesLib)
AppleSkin
AquaCulture 2
Aquamirae
ArchitecturyAPI
Artifacts
AttributeFix
Balm(Forge Edition)
Baubley Heart Cannisters
Blue Skies
Botarium
Caelus API (Forge/NeoForge)
Cloth Config API
Clumps
Comforts
Cosmetic Armor Reworked
Create
CreativeCore
Curios API
Divine RPG
Enchanting Infuser
ESSENTIAL Mod
FerriteCore
Gateways to Eterniety
GeckoLib
Handcrafted
Harvest with ease
Illager Invasion
Iron's Spells
Jade
JourneyMap
Kiwi
KleeSlabs
Lios Overhauled Villages
Lithostitched
Loot Beams: Relooted!
Lootr
Macaw's Doors
Macaw's Fences and Walls
Macaw's Furniture
Macaw's Lights and Lamps
Macaw's Roofs
Macaw's Trapdoors
Macaw's Windows
MES - Moog's End Structures
ModernFix
Nature's Compass
Not Enough Animations
Nullscape
Nyf's Quivers
Obscure API
ObsidianUI
OctoLib
Patchouli
Placebo
PlayerAnimator
PlayerRevive
Polymorph
Puzzles Lib
Relics
Reliquary Reincarnations
Resourceful Config
Resourceful Lib
Simply Swords
Snow! Real Magic!
Sophisticated Backpacks
Sophisticated Core
Structory
Structure Gel API
SuperMartijn642's Config Lib
SuperMartijn642's Core Lib
Sushi Go Crafting
Tarot Cards
Terralith
The Twilight Forest
Titanium
Trash Cans
Traveler's Titles
Twigs
Visual Workbench
Waystones
YUNG's API
YUNG'S Better Desert Temples
YUNG'S Better Dungeons
YUNG'S Better End Island
YUNG'S Better Jungle Temples
YUNG'S Better Mineshafts
YUNG'S Better Nether Fortresses
YUNG'S Better Ocean Monuments
YUNG'S Better Strongholds
YUNG'S Better Witch Huts
YUNG'S Bridges
YUNG'S Extras

### Additional Context

I think it might also help to know I got the modpack directly from a friend, as they created the modpack and ran it.
I ran this through CurseForge.

I thank you for taking your time to look at this!

**Comment by djnifos** [comment]
Another very similar one, this is in ATM9 https://mclo.gs/b5NxMJU

**Comment by Kaloyan501** [comment]
Are you using the Create mod? The Create mod comes with Flywheel inside it now, so you don't need to download the Flywheel .jar file separately.

**Comment by Kaloyan501** [comment]
Yeah, you're using Create, just remove the Flywheel jar from the mods directory and you should be ready to go!

**Comment by IThundxr** [comment]
> Yeah, you're using Create, just remove the Flywheel jar from the mods directory and you should be ready to go!

Thats not the issue here.

**Comment by Proxwian** [comment]
Any update here? Issue still persist

**Comment by ShadeUwU** [comment]
I've had the same issue for a year or so. Legit can't play a bunch of modpacks that have create.
Here's my crash log: https://pastebin.com/3w613GvA
And here's my Crash Message Screen
![image](https://github.com/user-attachments/assets/a95ba902-b53b-45e6-8e7f-06dbb30beb1d)
If anyone has any workaround please let me know. This is the most recent pack of [ATM9SKY](https://www.curseforge.com/minecraft/modpacks/all-the-mods-9-to-the-sky)

**Comment by IThundxr** [comment]
This is caused due to the `~`'s in your file path to the instance, forge doesn't properly load JiJ'd mods if the path to them has a tilde in them. Nothing can be done on flywheel's side

**Comment by ShadeUwU** [comment]
> This is caused due to the `~`'s in your file path to the instance, forge doesn't properly load JiJ'd mods if the path to them has a tilde in them. Nothing can be done on flywheel's side

Is this something that I can edit manually by simply removing the `~` from the path? or will this cause issues with mods looking into the incorrect folders?

**Comment by IThundxr** [comment]
> > This is caused due to the `~`'s in your file path to the instance, forge doesn't properly load JiJ'd mods if the path to them has a tilde in them. Nothing can be done on flywheel's side
> 
> Is this something that I can edit manually by simply removing the `~` from the path? or will this cause issues with mods looking into the incorrect folders?

You should just create a new instance under a different folder and copy your saves, mods, configs and anything else important over

**Comment by TropheusJ** [comment]
the `~` is in your user folder name. You will have to put everything somewhere outside of it.
Changing your user folder name is basically impossible without reinstalling windows.

**Comment by DadamaldaDad** [comment]
Don't you need Creste 6.0?
